notes-webapp/src/main/webapp/vue-app/notes-rich-editor/components/NoteEditorTopBar.vue

```diff
@@ -107,6 +108,7 @@
               width="36"
               min-width="36"
               height="36"
+              :aria-label="$t('notes.metadata.close.drawer')"
```

Comment:
we need to make the close button of properties drawer accessible not the close button of the editor on mobile

Reply:
yes, you're right, but we don't have such option in the drawer